tests/subsys/bluetooth/fast_pair/crypto/Kconfig

```diff
@@ -11,3 +11,8 @@ endmenu
 menu "Zephyr"
 source "Kconfig.zephyr"
 endmenu
+
+config BT_FAST_PAIR_TEST_CRYPTO_EXT
+	bool "Test crypto for Fast Pair extensions"
+	help
+	  Test crypto functions for Fast Pair extensions.
```

Comment:
I think this is no longer necessary as crypto implementations for extensions are resolved at the link time (If the test project in the main.c file references a given crypto API it should not be optimized by the linker)